### ISSUE TITLE
Unable to initialize JNetPCap

### ISSUE THREAD

**Issue by xmjie** [iadgov/GRASSMARLIN#11]
"Unable to initialize JNetPCap,packet capture functionality will be disabled",why?How I can solve this problem?

**Comment by iadgovuser5** [comment]
What version are you trying to run?
Are you installing from an installer or are you trying to build from source? 
What Operating System are you running on?

**Comment by iadgovuser5** [comment]
3.2 and 3.2.1 will produce this error message if GrassMarlin cannot find the native JNetPCap library in your working directory.

**Comment by xmjie** [comment]
The version is 3.2,and I don't find an installer,I tried to run by runing build-ant.xml.The operating system is windows.And lib has jnetpcap.jar.

**Comment by xmjie** [comment]
The version is 3.2,and I don't find an installer. I tried to run by running build-ant.xml, then running GrassMarlin.jar. The operating system is windows.And lib has jnetpcap.jar.

**Comment by iadgovuser5** [comment]
I would recommend downloading and running the v3.2.1 installer for your platform. The installers can be found under Releases on the main page or at [https://github.com/iadgov/GRASSMARLIN/releases/tag/v3.2.1](url).

**Comment by JsHuang** [comment]
Using the release windows installer(GrassMarlin-64.msi), I still encountered this problem .
My OS is Windows10 x64. I wonder if this has anything to do with the jdk version? My JDK version is JDK11.

**Comment by Canada-Lynx** [comment]
Log shows "Unable to initialize JNetPCap, packet capture functionality will be disabled", but the native JNetPCap library is just in the directory "GM3\lib\jnetpcap.jar". It seems that something wrong happens when the code :
        "System.loadLibrary("jnetpcap")"  (Line 84 in "GM3\src\util\Launcher.java") 
is executed. 
Have you solved this problem yet? Can you tell me what I should do to solve this problem? Thank you very much. 
![GrassMarlin_log](https://user-images.githubusercontent.com/41562420/58537870-6bc5bd00-8226-11e9-8a8d-b4d0d321f98e.png)
![Launcher](https://user-images.githubusercontent.com/41562420/58538347-83ea0c00-8227-11e9-9a25-dc80170e36af.png)

**Comment by iadgovuser22** [comment]
Hi,
Can you describe your work environment and indicate what version do you have installed?

Thanks

**Comment by Canada-Lynx** [comment]
Thank you so much for your reply!
I think the source files may be v3.2 or a higher version since they were downloaded in August 2017. 
They were compiled with java version "1.8.0_212" and run in win10_x64.

**Comment by iadgovuser22** [comment]
Hi,

I'm assuming you are running GM from source. If this is the case, have you tried using the msi package to install the pre-compiled code? I'll have to look deeper into it.

Thanks

**Comment by Canada-Lynx** [comment]
Yes. I have tried using the msi package GrassMarlin-64.msi to install the pre-compiled code but the same log appears like the picture. It seems that something is wrong with the configuration or environment but I don't know how to fix it.
Thanks anyway.

![msi64](https://user-images.githubusercontent.com/41562420/58742709-5e901480-8458-11e9-926f-d5f37f6f92d0.png)

**Comment by ruby-wizard** [comment]
I am also getting this error. I just installed the GrassMarlin-Full-x64. And after running the installer and opening up GM, the error appears. I confirmed that I do have WinPCap installed as well

**Comment by ruby-wizard** [comment]
**[SOLVED]** I had 2 Java installs, a newer 210 and the 92 (which is from the GrassMarlin install) GrassMarlin default pointed to the wrong Java install. Removing both and only installing the 92 worked for me!
